src/main/kotlin/nl/info/zac/app/klant/model/klant/IdentificatieType.kt

```diff
@@ -7,5 +7,8 @@ package nl.info.zac.app.klant.model.klant
 enum class IdentificatieType {
     BSN,
     VN,
-    RSIN
+    RSIN;
```

Comment:
An alternative way to the methods `isBsn()` and `isKvK()` would be to define these as properties and set them on creation of the enum instances, ie:

```kotlin
enum class IdentificatieType(
    val isBsn: Boolean = false,
    val isKvK: Boolean = false
) {
    BSN(isBsn = true),
    VN(isKvK = true),
    RSIN(isKvK = true)
}
```

Reply:
Not to say that this is "better", but I, personally, prefer this implementation.